src/one/nio/serial/Repository.java

```diff
@@ -82,6 +82,7 @@
         addBootstrap(new DateSerializer());
         addBootstrap(new ClassSerializer());
         addBootstrap(new BitSetSerializer());
+        addBootstrap(new UuidSerializer());
```

Comment:
The order of bootstrap serializers is important. So, the new serializers should be added at the very bottom for the backward compatibility reasons.